sdk-python/examples/correlated_event/example_correlated_event.py

```diff
@@ -0,0 +1,70 @@
+import asyncio
+import logging
+from pathlib import Path
+
+import littlehorse
+from littlehorse.config import LHConfig
+from littlehorse.model import VariableType, VariableMutationType
+from littlehorse.worker import LHTaskWorker
+from littlehorse.workflow import WorkflowThread, Workflow
+from littlehorse.model import PutExternalEventDefRequest, CorrelatedEventConfig
+
+logging.basicConfig(level=logging.INFO)
+
+ASK_FOR_NAME = "ask-for-name"
+GREET = "greet"
+EXT_EVENT = "name-event"
+
+
+def get_config() -> LHConfig:
+    config = LHConfig()
+    config_path = Path.home().joinpath(".config", "littlehorse.config")
+    if config_path.exists():
+        config.load(config_path)
+    return config
+
+
+def get_workflow() -> Workflow:
+    def my_entrypoint(wf: WorkflowThread) -> None:
+        wf.execute(ASK_FOR_NAME)
+        ext_event_output = wf.wait_for_event(
+            EXT_EVENT,
+            correlation_id="some-correlation-id",
+            mask_correlation_id=True
+        )
+
+        name = wf.add_variable("name", VariableType.STR)
+        wf.mutate(name, VariableMutationType.ASSIGN, ext_event_output)
```

Comment:
Super small nit pick: we're trying to get people to use `name.assign(ext_event_output)` rather than `wf.mutate(...)`. The external event example was written before this so it wasn't available. I'd like our examples to start using the "recommended" patterns.

(Also note: you'll need to remove the unused import for `VariableMutationType` above or the linter will get grumpy again)